policy/release/cve.rego

```diff
@@ -42,28 +44,30 @@ deny contains result if {
 #   "warn_cve_security_levels". The available levels are critical, high,
 #    medium, and low.
 # custom:
-#   short_name: found_non_blocking_cve_vulnerabilities
+#   short_name: cve_warnings
 #   failure_msg: Found %d non-blocking CVE vulnerabilities of %s security level
 #   collections:
 #   - minimal
+#
 warn contains result if {
 	some level, amount in _non_zero_levels("warn_cve_security_levels")
 	result := lib.result_helper_with_term(rego.metadata.chain(), [amount, level], level)
 }
 
 # METADATA
-# title: Missing CVE scan results
+# title: CVE scan results found
 # description: >-
 #   The clair-scan task results have not been found in the SLSA Provenance
 #   attestation of the build pipeline.
```

Comment:
Just a note here, that we're looking at introducing the CRDA task in some pipeline configurations, so there could be multiple tasks (or different tasks) providing vulnerability results in the future.

Reply:
I think it's clear enough from the description and the solution text that it's clair doing the scanning. I'd rather keep the back end detail out of the title for now. Introducing another source of CVEs will be a non-trivial change for the rego logic here I think, so let's revisit the rule titles then.